### ISSUE TITLE
Update Introduction Section to Include Citation

### ISSUE THREAD

**Issue by niazim3** [JacquesCarette/Drasil#680]
(Propagation originating from smiths/caseStudies#37).

As visible in [the changes made for PR #642](https://github.com/JacquesCarette/Drasil/pull/642/files), specifically in the [Introduction.hs file](https://github.com/JacquesCarette/Drasil/pull/642/files#diff-c3aeef9a4cc056cbecb00d767d7e20a1), the introduction needs to cite ParnasAndClements1986.

Why this wasn't a quick change was because the `Contents` that need to be updated (i.e. `developmentProcessParagraph` from Introduction.hs) needs to be passed in a `ReferenceDB` specific to the examples (because attempting to avoid to do so (through `rdb [] [] [] [] [] []`) resulted in errors like the following: 
```
cd build/SWHS && stack exec swhs
swhs.EXE: Change: parnasClements1986 referencing information not found in Change Map
CallStack (from HasCallStack):
  error, called at .\Language\Drasil\Reference.hs:151:24 in drasil-0.0.9-EhO8nEtQjIEKb093SzEDf2:Language.Drasil.Reference
```
), which is used only by `purposeOfDoc`, which is used by `introductionF` (not used anywhere) and `mkIntroSec` (which is from DocumentLanguage.hs) -> `mkSections` -> `mkDoc` -> Examples; making it a "deeper" issue to work on.

**Comment by JacquesCarette** [comment]
Doesn't this just mean that the `ReferenceDB` for SWHS did not have parnasClements1986 in it, and it is needed?  [The hack of passing in an empty DB clearly shows that that hack is a non-starter, which is a good thing].

**Comment by szymczdm** [comment]
Yep, it looks like the issue is exactly that. The RefDB is empty when it shouldn't be (someone, somewhere, is attempting to look up that `parnasClements1986` reference), so it rightfully fails.

**Comment by niazim3** [comment]
Since `introductionF` is not used anywhere, can it be removed? Or should it be updated for the changes made in commit 6e5ca8c (i.e. `purposeOfDoc` expects a `ReferenceDB` as an argument)?

**Comment by JacquesCarette** [comment]
If none of our examples use it, then I don't think `introductionF` is useful, and it should indeed be removed.

**Comment by niazim3** [comment]
Closes with merge of #690.
